homeassistant/components/bosch_alarm/binary_sensor.py

```diff
@@ -0,0 +1,73 @@
+"""Support for Bosch Alarm Panel binary sensors."""
+
+from __future__ import annotations
+
+from bosch_alarm_mode2 import Panel
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import BoschAlarmConfigEntry
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant | None,
+    config_entry: BoschAlarmConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up binary sensors for alarm points and the connection status."""
+    panel = config_entry.runtime_data
+
+    async_add_entities(
+        PointSensor(panel, point_id, config_entry.unique_id or config_entry.entry_id)
+        for point_id in panel.points
+    )
+
+
+PARALLEL_UPDATES = 0
+
+
+class PointSensor(BinarySensorEntity):
+    """A binary sensor entity for a point in a bosch alarm panel."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+
+    def __init__(self, panel: Panel, point_id: int, unique_id: str) -> None:
+        """Set up a binary sensor entity for a point in a bosch alarm panel."""
+        self.panel = panel
+        self._attr_unique_id = f"{unique_id}_point_{point_id}"
+        self._point = panel.points[point_id]
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self._attr_unique_id)},
+            manufacturer="Bosch Security Systems",
+            name=self._point.name,
+            model=panel.model,
+            sw_version=panel.firmware_version,
+            via_device=(
+                DOMAIN,
+                unique_id,
+            ),
+        )
```

Comment:
I think we can omit the things like model and such, as that's info about the panel, not the point itself